tick/plot/plot_stem.py

```diff
@@ -1,89 +1,138 @@
-    fig.title_text_font_size = "12pt"
+    fig.title.text_font_size = "12pt"
     if title is not None:
-        fig.title = title
-    if show:
-        bk.show(fig)
-        return None
-    else:
-        return fig
+        fig.title.text = title
+    return fig
 
 
-def stems(ys: list, titles: list = None, show=True, sync_axes=True):
-    """Several stem plots with synchronized axes
+def __stems_bokeh(ys: list, titles: list, sync_axes, fig_size):
+    from bokeh.plotting import gridplot
+
+    figs = []
+    x_range = None
+    y_range = None
+    for idx, (y, title) in enumerate(zip(ys, titles)):
+        fig = __stem_bokeh(y, title=title, x_range=x_range, y_range=y_range,
+                           fig_size=fig_size)
+        figs.append(fig)
+        if idx == 0 and sync_axes:
+            x_range = fig.x_range
+            y_range = fig.y_range
+    return gridplot([[e] for e in figs])
+
+
+def stems(ys: list, titles: list = None, sync_axes: bool = True,
+          rendering: str = 'matplotlib', fig_size: tuple = None):
+    """Plot several stem plots using either matplotlib or bokeh rendering.
+    Axes can be synchronized, which means that all xlim and ylim are the
+    same for matplotlib rendering, while zooming is done simultaneously on all
+    stem plots for bokeh rendering.
 
     Parameters
     ----------
-    ys : `list` of `np.ndarray`
-        A list of numpy arrays to be plotted
-
-    titles : `list` of `str`
-        The titles of each plot
+    ys : `list` of `numpy.array`
+        A list of numpy arrays to be plotted using stem plots
 
-    show : `bool`, default=`True`
-        if True, show the plot. Use False if you want to superpose several
-        plots
+    titles : `list` of `str`, default=`None`
+        The titles of each stem plot
 
     sync_axes : `bool`, default=`True`
-        If True, the axes of the stem plot are synchronized
+        If `True`, axes can be synchronized, which means that for
+        matplotlib rendering, the
+        xlim and ylim are the same, while for bokeh rendering, this means that
+        zooming is done simultaneously on all stems.
+
+        If True, the axes of the stem plot are synchronized (available only
+        with ``rendering='bokeh'``
+
+    rendering : {'matplotlib', 'bokeh'}, default='matplotlib'
+        Rendering library. 'bokeh' might fail if the module is not installed.
+
+    fig_size: `tuple`, default=`None`
+        Figure size. Default is (8, 2.5) for matplotlib rendering and
+        (600, 200) for bokeh rendering.
 
     Returns
     -------
-    output : `bk.grid_plot`
-        A grid plot object
+    output : `bokeh.models.layouts.Column`, `matplotlib.pyplot.Figure` or `None`
+        Depending on the rendering type, returns a ``bokeh`` or a ``matplotlib``
+        object containing the layout of the plot, or `None` otherwise
     """
-    import bokeh.plotting as bk
+    from bokeh.plotting import show as bk_show
```

Comment:
This should be imported in the bokeh part of the code. This idea is that tick should be able to run without bokeh installed